src/Components/Footer/Footer.js

```diff
@@ -1,9 +1,133 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import styled, { css } from "styled-components";
 
 class Footer extends Component {
   render() {
-    return <div></div>;
+    return (
+      <FooterContents>
+        <FooterContainer>
+          <FooterLeftTitle>
+            <ul>
+              <span>© Kinfolk 2020</span>
+              <li>
+                <Link to="/">Terms</Link>
+              </li>
+              <li>
+                <Link to="/productList">Subscribe</Link>
+              </li>
+            </ul>
+          </FooterLeftTitle>
+          <div>
+            <FooterSnsTitle>
+              <li>
+                <a
+                  alt="FactbookLink"
+                  href="https://www.facebook.com/kinfolkmag"
+                >
+                  Facebook
+                </a>
+              </li>
+              <li>
+                <a alt="TwitterLink" href="https://twitter.com/kinfolkmag">
+                  Twitter
+                </a>
+              </li>
+              <li>
+                <a
+                  alt="InstargramLink"
+                  href="https://www.instagram.com/kinfolk/"
+                >
+                  Instargram
+                </a>
+              </li>
+            </FooterSnsTitle>
+          </div>
+          <span>
+            <a href="https://www.madebysix.com/">
+              Website design & development by Six
+            </a>
+          </span>
+        </FooterContainer>
+      </FooterContents>
+    );
   }
 }
 
+const footerHover = css`
+  &:link {
+    color: black;
+  }
+
+  &:visited {
+    color: black;
+  }
+
+  &:active {
+    color: black;
+  }
+
+  &:hover {
+    transition: ease 250ms color;
+    color: #999999;
+  }
+`;
+
+const listMargin = css`
+  margin-right: 35px;
+`;
+
+const setSize = (width, height) => css`
+  width: ${width};
+  height: ${height};
+`;
```

Comment:
```suggestion
const setSize = (width, height = null) => css`
  width: ${width};
  height: ${height};
`;
```
이렇게 기본값 설정해두는 건 어때요?

Reply:
wow 😲..22..

Reply:
기본값 설정이라니 너무 좋은데요 진짜